d3a_interface/device_validator.py

```diff
@@ -113,6 +113,16 @@ def validate_load_device(**kwargs):
             {"misconfiguration": [f"daily_load_profile and all or one [hrs_per_day, hrs_of_day, "
                                   f"avg_power_W] can't be set together."]})
 
+    if "daily_load_profile" in kwargs and kwargs["daily_load_profile"] is not None:
+        print()
+        if isinstance(kwargs["daily_load_profile"], str):
+            _validate_energy_profile(ast.literal_eval(kwargs["daily_load_profile"]))
```

Comment:
Can you add this `validate_load_device` validator in [DefinedLoadStrategy](https://github.com/gridsingularity/d3a/blob/master/src/d3a/models/strategy/predefined_load.py#L79) where `daily_load_profile` could be a profile path.

Reply:
Sorry, i dont get it. 
In the d3a the `read_arbitrary_profile` is dealing with reading and validating the daily_load_profile. 
This is why i suggested to you in one of your last PRs in this repo to move this function from the d3a to here. 
But you had a good reason not to. But i can't remember. Can you?

Reply:
I can't recall it either, well then skip it for now and we can look into later when needed.